.github/workflows/migration-helper01.yml

```
@@ -17,6 +17,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)